metricbeat/metricbeat.reference.yml

```diff
@@ -477,7 +477,6 @@ metricbeat.modules:
 
 #------------------------------- vSphere Module ------------------------------
 - module: vsphere
-  metricsets: ["datastore", "host", "virtualmachine"]
```

Comment:
We should keep all metricsets in the reference file and also add `enabled: true`.

I assume there is no `config.reference.yml` in vshpere so we probably have to add one.